podenv/pod.py

```diff
@@ -167,7 +167,8 @@ def setupVolumes(volumes: Volumes) -> None:
     existingVolumes: Set[str] = set()
     if volumesList:
         for volume in volumesList:
-            existingVolumes.add(volume['name'])
+            volume_name = volume.get('Name') or volume.get('name')
+            existingVolumes.add(volume_name)
```

Comment:
hmm, mypy should have complained that volume_name is an Optional[str] ... 
Could you please add a:
```python
if not volume_name:
    raise RuntimeError(f"Failed to parse volume: {volume}")
```